FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -2,6 +2,35 @@
   "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    "location": {
+        "type": "string",
+        "defaultValue": "East US",
+        "allowedValues": [
+            "Australia East",
+            "Brazil South",
+            "Canada Central",
+            "Central US",
+            "East Asia",
+            "East US",
+            "East US 2",
+            "France Central",
+            "Germany West Central",
+            "Japan East",
+            "Korea Central",
+            "North Central US",
+            "North Europe",
+            "Norway East",
+            "South Africa North",
+            "South Central US",
+            "UK South",
+            "West Europe",
```

Comment:
Just want to confirm that Container Apps can be deployed to these regions?

Reply:
Sure, I have manually deployed on those regions